netlify.toml

```diff
@@ -2,6 +2,7 @@
   base = "src/frontend/next/"
   publish = "out"
   command = "npm run build"
+  ignore = "/bin/false"
```

Comment:
Let's add a comment above this line linking to this PR, which documents why we do this.